src-opam/dockerfile_distro.ml

```diff
@@ -109,14 +109,14 @@ let distro_status (d:t) : status = match d with
   | `Ubuntu ( `V12_04 | `V14_04 | `V15_04 | `V15_10 | `V16_04 | `V16_10 | `V17_04 | `V17_10 | `V18_10 | `V19_04 | `V19_10 ) -> `Deprecated
   | `Ubuntu `LTS -> `Alias (`Ubuntu `V20_04)
   | `Ubuntu `Latest -> `Alias (`Ubuntu `V20_10)
-  | `Cygwin `V20H2 -> `Active `Tier3
+  | `Cygwin `V21H1 -> `Active `Tier3
   | `Windows (_, _) -> `Active `Tier3
```

Comment:
We need a `win10_update_supported` function for this. 1903, for example, ended support in November. I presume that the 1809 Docker images are receiving LTSC updates, but that's worth checking (by looking at the build number and checking the Latest Cumulative Update info from Microsoft). It may anyway be worth a having that function return a level of support - i.e. ``[`Consumer | `Enterprise | `LTSC]`` (see the [helpful table on Wikipedia](https://en.wikipedia.org/wiki/Windows_10_version_history#Channels)).

Reply:
I did not consider the level of support, but I think I got a sensible fix around this.